ios/FluentUI.Demo/FluentUI.Demo/Demos/ThemingDemoController.swift

```diff
@@ -85,21 +84,19 @@ class ThemingDemoController: DemoController {
 
         addRow(items: [customThemeButtonPrimary.view, customThemeButtonSecondary.view, customThemeButtonGhost.view], itemSpacing: 20)
 
+//        let overridingTheme = CustomStyle()
```

Comment:
Likely related to this comment?
https://github.com/microsoft/fluentui-apple/pull/891#discussion_r801307242